autosklearn/ensemble_building/builder.py

```diff
@@ -899,8 +914,8 @@ def loss(self, run: Run, kind: str = "ensemble") -> float:
                 solution=targets,
                 prediction=predictions,
                 task_type=self.task_type,
-                metrics=[self.metric],
-            )[self.metric.name]
```

Comment:
Still only getting a single loss per run?

Reply:
Yes, so far only one metric. I'd leave this to you to change this when you update ensemble builder to prune the library based on multiple metrics.